python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -272,6 +281,17 @@ class PiffPsfDeterminerTask(BasePsfDeterminerTask):
     ConfigClass = PiffPsfDeterminerConfig
     _DefaultName = "psfDeterminer.Piff"
 
+    def __init__(self, config, schema=None, **kwds):
+        BasePsfDeterminerTask.__init__(self, config, schema=schema, **kwds)
+
+        piffLoggingLevels = {
+            0: logging.CRITICAL,
+            1: logging.WARNING,
+            2: logging.INFO,
+            3: logging.DEBUG,
+        }
+        self.log.setLevel(piffLoggingLevels[self.config.piffLoggingLevel])
```

Comment:
This looks like it's setting the level for PiffPsfDeterminer, not piff?  Does this make the log messages [here](https://github.com/lsst/meas_extensions_piff/blob/main/python/lsst/meas/extensions/piff/piffPsfDeterminer.py#L305) and [here](https://github.com/lsst/meas_extensions_piff/blob/main/python/lsst/meas/extensions/piff/piffPsfDeterminer.py#L308) subject to this config setting?

Reply:
Can you instead try something like:
```python
piffLogger = logging.getLogger(f'{self.log.name}.piff')
piffLogger.setLevel(...)
```
and then use `piffLogger` when calling `piffResult.fit` ?

Reply:
Ah great point!